### PR TITLE
colorized dbt output

### PR DESCRIPTION
**Pull request by drewbanin**
TODO:
- [x] Test on windows
- [x] Add option to turn this off?

### CODE REVIEW

File: dbt/config.py [hunk 1]
```diff
@@ -25,14 +25,8 @@ def read_profile(profiles_dir):
...
-def send_anonymous_usage_stats(profiles_dir):
-    config = read_config(profiles_dir)
-
-    if config is not None \
-       and not config.get("send_anonymous_usage_stats", True):
-        return False
-
-    return True
+def send_anonymous_usage_stats(config):
+    return config.get('send_anonymous_usage_stats', True)
```
Comment (cmcarthur) [inline dbt/config.py]: @drewbanin I'm not sure this works. There are some unit test failures in circle & appveyor

File: dbt/runner.py [hunk 2]
```diff
@@ -19,15 +17,17 @@
 import dbt.schema
 import dbt.graph.selector
 import dbt.model
+import dbt.printer as printer
```
Comment (cmcarthur) [inline dbt/runner.py]: can you change this file to use the globally qualified name?

File: dbt/printer.py [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+    if stats['error'] == 0:
+        message = green('Completed successfully')
+    else:
+        message = red('Completed with errors')
+
+    stats_line = "Done. PASS={pass} ERROR={error} SKIP={skip} TOTAL={total}"
+    stats_line = stats_line.format(**stats)
+
+    return "{}\n{}".format(message, stats_line)
```
Comment (cmcarthur) [inline dbt/printer.py]: love that all this got pulled out of runner

File: dbt/printer.py [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+    global USE_COLORS
+    USE_COLORS = True
+
+def get_timestamp():
+    return time.strftime("%H:%M:%S")
+
+
+def color(text, color_code):
+    if USE_COLORS:
+        return "{}{}{}".format(color_code, text, colorama.Style.RESET_ALL)
```
Comment (cmcarthur) [inline dbt/printer.py]: @drewbanin is it possible to create an enumeration of available colors here, like:

```
RED_FOREGROUND = colorama.Fore.RED
GREEN_FOREGROUND = colorama.Fore.GREEN
YELLOW_FOREGROUND = colorama.Fore.YELLOW
...
```

and then reference those constants directly?

I'm thinking that would make it easy to override the colors when using dbt as a library

**Comment by cmcarthur** [comment]
:shipit: !!

**Comment by drewbanin** [comment]
fixes https://github.com/fishtown-analytics/dbt/issues/357

**Pull request merged**